content/posts/easystats_threeyears.Rmd

```diff
@@ -0,0 +1,144 @@
+
+- [**datawizard**](https://easystats.github.io/datawizard): 🧙 The most recent addition to the ecosystem! This package form the data wrangling backend for the entire ecosystem and can be just what you are looking for if you want to do *tidyverse*-like data wrangling but wish to avoid adding dependencies to your project.
+
+We then have specialized packages to extract additional information from statistical models.
+
+- [**bayestestR**](https://easystats.github.io/bayestestR): 👻 A package that provides utility tools for manipulating and visualizing Bayesian statistical models.
+
+- [**effectsize**](https://easystats.github.io/effectsize): 🐉 A one-stop solution for computing pretty much every effect size known to the mankind, well, almost every. 😅
+
+- [**correlation**](https://easystats.github.io/correlation): 🔗 A package dedicated for computing and visualizing probably the most comprehensive suite of correlations.
+
+- [**modelbased**](https://easystats.github.io/modelbased): 📈 A utility package to work with predictions from statistical models, and can be used to compute and visualize marginal means or contrast analyses.
+
+The information about statistical models are extracted and aggregated in the following two packages:
+
+- [**performance**](https://easystats.github.io/performance): 💪 Computes, analyzes, and tests the performance of statistical model(s).
+
+- [**parameters**](https://easystats.github.io/parameters): 📊 Extracts a comprehensive dataframe of almost any statistical model's parameters and provides helpers to present them in elegant tables and plots.
+
+Finally, the two high-level packages utilize other packages to communicate results from statistical analyses, either graphically or textually.
+
+- [**see**](https://easystats.github.io/see): 🎨 This package forms the visualization backend for the entire ecosystem and interfaces with *ggplot2* to support plotting methods for various *easystats* objects.
+
+- [**report**](https://easystats.github.io/report): 📜 🎉 This packages provides an automated way to create text reports detailing results from statistical analyses.
+
+Except for *datawizard*, *report*, and *modelbased* packages, which are still in their teenage years, the rest of the packages are legally permitted to drive your statistical analyse as they have been published in peer-reviewed journals^[You can use `citation()` function to see reference for each package.] and enjoy a regular health checkup with a large suite of unit tests.
+
+### Soaring Like an Eagle!
+
+To our utmost delight, these packages have already been downloaded **over 6 million times**! 🤯
+
+It has been an overwhelming experience to see the warm welcome this project has received in the `#rstats` community on Twitter and LinkedIn.
+
+### "Let brotherly love continue"
```

Comment:
As much as these titles are very elegant and poetic (as the rest of the text is), I wonder if more straightforward points would not be better for the people that tend to skim through blog posts just by reading titles. Titles like "It has become very popular" "It is used by many other packages" etc. convey the info perhaps more clearly?

Reply:
I don't know. I feel like titles like these might actually attract users to read the section. 

If I read a section header like, _"It has become very popular"_, I don't see the point of reading the section because I already know what you are going to say. But if I read _"Let brotherly love continue"_, you have piqued my curiosity. But maybe this is very idiosyncratic of just me. 

If others feel the same way you do, happy to fall back to the original headers.

Reply:
A Compromise?

`Brotherly love: *easystats* is used in many packages`

Reply:
– Hebrews 13:1-2 😅 

Fine by me.